libs/shared-atomic-design-components/src/lib/molecules/ProfileCard.tsx

```
@@ -45,57 +45,55 @@ const UserLanguages = ({ languages }: UserLanguagesProps) => {
+const ProfileCard = ({
   profile: { id, avatar, fullName, location, languages },
   tags,
   subheader,
   linkTo,
   toggleFavorite,
   isFavorite,
-}: NewProfileCardProps) {
+}: ProfileCardProps) => {
   const handleFavoriteClick = (e: React.MouseEvent) => {
     e.preventDefault()
     toggleFavorite && toggleFavorite(id)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)